pkg/admin_client/client.go

```diff
@@ -0,0 +1,83 @@
+package admin_client
```

Comment:
what do you think about calling this features_client? or are you anticipating using this for more admin only (and not features-related) things?

Reply:
My thinking was it could be used for things non feature related since it's just making http calls (without a generated binding like the other clients). I think I'll know more about how to organize things with the next part of this

Reply:
gotcha, makes sense!

Reply:
I think it does make sense to rename this for now, so I'm going to push that as well